control-plane/cmd/kafka-source-controller/main.go

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2022 The Knative Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package main
+
+import (
+	"context"
+	"log"
+
+	"knative.dev/pkg/injection"
+	"knative.dev/pkg/signals"
+
+	"knative.dev/pkg/configmap"
+	"knative.dev/pkg/controller"
+	"knative.dev/pkg/injection/sharedmain"
+
+	"knative.dev/eventing-kafka-broker/control-plane/pkg/config"
+	"knative.dev/eventing-kafka-broker/control-plane/pkg/reconciler/consumer"
+	"knative.dev/eventing-kafka-broker/control-plane/pkg/reconciler/consumergroup"
+	sourcev2 "knative.dev/eventing-kafka-broker/control-plane/pkg/reconciler/source/v2"
+)
+
+const (
+	component = "kafka-broker-controller"
+)
+
+func main() {
+
+	sourceEnv, err := config.GetEnvConfig("SOURCE")
+	if err != nil {
+		log.Fatal("cannot process environment variables with prefix SOURCE", err)
+	}
```

Comment:
I'm removing this unused source env in https://github.com/knative-sandbox/eventing-kafka-broker/pull/2093 so we can remove this and the associated env variables in the `kafka-source-controller` deployment.